pipeline/07-export.R

```diff
@@ -620,7 +635,7 @@ for (town in unique(assessment_pin_prepped$township_code)) {
   num_head <- 6 # Number of header rows
   pin_row_range <- (num_head + 1):(nrow(assessment_pin_filtered) + num_head)
   pin_row_range_w_header <- c(num_head, pin_row_range)
-  pin_col_range <- 1:67 # Don't forget the two hidden rows at the end
+  pin_col_range <- 1:68 # Don't forget the two hidden rows at the end
```

Comment:
This coming year I'd like to actually implement https://github.com/ccao-data/model-res-avm/issues/221 so that we don't have to do this annoying index shifting every time we add or remove a column.